### DISCUSSION TITLE
Reviews on weather camera

### DISCUSSION THREAD

**Discussion by f4gqk** [poblabs/weewx-belchertown#787] [Q&A]
Hello,

I would like to have your opinion for the purchase of a camera dedicated to the weather (brand and model) and also to have your feedback and your settings.

- For example put photos every 5 minutes on an FTP server..... and the report with weewx Belchertown

Thanks

Sébastien
[www.meteo-guignes.fr](url)

**Comment by ChrisM510** [comment]
Well, I had set up my WeeWX/Belchertown to pull the last image from my Bloomsky unit, but then Bloomsky screwed everything up, and now the system isn't working anymore.  It was really nice to have when it was working.

**Reply by f4gqk** [reply]
Thank you for your answer.

The author of the Belchertown skin uses a blink outdoor. I will see in another message, how he integrates it but I agree on the principle that we only recover the last photo.

Sébastien

**Reply by poblabs** [reply]
I am using the blinkpy python module which authenticates to the blink service. It requests the camera to wake up, take a snapshot, then it downloads that image to my web server. I have that image in my weewx.conf for the skin to use. So for every time the photo updates, its from a new live image at that time.
